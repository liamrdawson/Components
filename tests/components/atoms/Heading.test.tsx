import React from 'react'
import { jsx } from '@emotion/react'
import {render} from '@testing-library/react'
import { Heading } from '../../../src'

test('Snapshot Test: Heading Level One', () => {
  const {container} = render(
    <Heading element="h1">Heading Level One</Heading>,
  )
  expect(container).toMatchSnapshot()
})

test('Snapshot Test: Heading Level Two', () => {
  const {container} = render(
    <Heading element="h2">Heading Level One</Heading>,
  )
  expect(container).toMatchSnapshot()
})

test('Snapshot Test: Heading Level Three', () => {
  const {container} = render(
    <Heading element="h3">Heading Level One</Heading>,
  )
  expect(container).toMatchSnapshot()
})

test('Snapshot Test: Heading Level Four', () => {
  const {container} = render(
    <Heading element="h4">Heading Level One</Heading>,
  )
  expect(container).toMatchSnapshot()
})
