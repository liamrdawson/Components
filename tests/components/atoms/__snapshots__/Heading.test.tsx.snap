// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test: Heading Level Four 1`] = `
<div>
  <h4
    class="css-1eiceoa-Heading"
  >
    Heading Level One
  </h4>
</div>
`;

exports[`Snapshot Test: Heading Level One 1`] = `
<div>
  <h1
    class="css-1eiceoa-Heading"
  >
    Heading Level One
  </h1>
</div>
`;

exports[`Snapshot Test: Heading Level Three 1`] = `
<div>
  <h3
    class="css-1eiceoa-Heading"
  >
    Heading Level One
  </h3>
</div>
`;

exports[`Snapshot Test: Heading Level Two 1`] = `
<div>
  <h2
    class="css-1eiceoa-Heading"
  >
    Heading Level One
  </h2>
</div>
`;
